docassemble/CivilActionCoverSheet/data/questions/Civil_Action_Cover_Sheet.yml

```diff
@@ -1,10 +1,17 @@
 ---
 include:
-  - docassemble.MAVirtualCourt:basic-questions.yml
+  - docassemble.AssemblyLine:al_package.yml
+  - docassemble.MassAccess:massaccess.yml
   - track_data.yml
 ---
 metadata:
   title: Civil Action Cover Sheet
+  authors:
+    - Maeve MacGlinchey
+    - Lance Godard
```

Comment:
I just realized this after testing sending an email to myself from the Motion to vacate marital home, but we do need the `interview_metadata` code block in interviews, otherwise we'll get a bunch of "None"s popping up in places that they shouldn't. 

I had one show up when I send an email to myself from the Vacate marital home, would probably show up on this on as well.

Reply:
I'm dumb, and there is an interview metadata in this interview. NM.